farmerbot/cmd/run.go

```diff
@@ -55,3 +72,15 @@ var runCmd = &cobra.Command{
 		return farmerBot.Run(cmd.Context())
 	},
 }
+
+func disclaimerPrompt() (string, error) {
+	prompt := promptui.Select{
```

Comment:
I Love this!! however let's see if we can just get by using fmt.Print and fmt.Scanf ^_^

Reply:
okay